packages/syntax-core/src/Chip/Chip.stories.tsx

```diff
@@ -106,10 +108,42 @@ const ChipIconInteractive = () => {
   );
 };
 
+const SyntaxIconInteractive = () => {
+  const [isSmSelected, setIsSmSelected] = useState(false);
+  const [isLgSelected, setIsLgSelected] = useState(false);
+  const handleSmChange = () => {
+    setIsSmSelected(!isSmSelected);
+  };
+  const handleLgChange = () => {
+    setIsLgSelected(!isLgSelected);
+  };
+
+  return (
+    <Box display="flex" gap={2} alignItems="center" flexWrap="wrap">
```

Comment:
@ajmooo fyi the Syntax icons within a badge are very tight to top and bottom
![image](https://github.com/Cambly/syntax/assets/127199/9aad1778-37db-4b1d-b2de-f2849d256fa6)

Reply:
updated: 
![image](https://github.com/Cambly/syntax/assets/15243713/a153ab66-bdf4-4ac6-a9fe-b68f424b784c)